subjects/ai/numpy/README.md

```diff
@@ -170,35 +186,58 @@ The goal of this exercise is to learn to access values of n-dimensional arrays e
        [1, 1, 1, 1, 1, 1, 1, 1, 1]], dtype=int8)
    ```
 
-3. Using **broadcasting** create the ouptu matrix starting from these two arrays:
+3. Using **broadcasting** create an output matrix based on the following two arrays:
 
    ```python
-   array_1 = np.array([1,2,3,4,5], dtype=int8)
+   array_1 = np.array([1,2,3,4,5], type=int8)
    array_2 = np.array([1,2,3], dtype=int8)
```

Comment:
The correct command should be `np.array([..], dytpe=np.int8)`

Reply:
done